ironfish/src/primitives/spend.ts

```diff
@@ -4,6 +4,8 @@
 
 import { Nullifier } from './nullifier'
 
+export const SPEND_SERIALIZED_SIZE_IN_BYTE = 388
```

Comment:
It feels a bit odd to define this alongside `Spend`, which doesn't contain most of the bytes that are in a spend as serialized on a transaction, but I'm not sure that it fits better anywhere else.

Reply:
Once TransactionBuilder is ready, we should be able to get rid of the use of constants.